### DISCUSSION TITLE
Postgresql driver

### DISCUSSION THREAD

**Discussion by makroxyz** [tenancy/tenancy#222] [Q&A]
PostgreSQL driver was present in 1.1-dev, but now repository is empty.
Why did you remove it? Is there any chance to have a release in the near future?

**Comment by ArlonAntonius** [comment]
Hi @makroxyz,

We're actively discussing and working on the PSQL Driver for Tenancy. We are debating a few critical points in the open PR #213. We'd love for you to join our discussions and share your views on the issues at hand.
We removed the 1.1-dev branch as we wanted to slightly restructure things for ourselves and a release with PSQL will be done once the driver is done.
